images/alpine/cloudbuild.yaml

```diff
@@ -1,19 +1,40 @@
 steps:
-  - name: gcr.io/cloud-builders/docker
-    args:
-    - build
```

Comment:
Will you please preserve this method of declaring lists?

Reply:
Done.